newdle/client/src/client.js

```diff
@@ -112,9 +112,13 @@ class Client {
     });
   }
 
-  getParticipant(newdleCode, participantCode) {
-    const params = {code: newdleCode, participant_code: participantCode};
-    return this._request(flask`api.get_participant`(params), {anonymous: true});
+  getParticipant(newdleCode, participantCode, me) {
+    if (me) {
```

Comment:
I'd use `participantCode === null` for that, but nothing wrong with being explicit.